packages/common/src/scopeSupportFacets/scopeSupportFacets.types.ts

```diff
@@ -26,20 +26,26 @@ const scopeSupportFacets = [
   "class",
   "class.iteration.document",
   "class.iteration.block",
+
   "className",
   "className.iteration.document",
   "className.iteration.block",
+
+  "anonymousFunction",
+
   "namedFunction",
-  "namedFunction.method",
-  "namedFunction.method.iteration.class",
   "namedFunction.iteration",
   "namedFunction.iteration.document",
-  "anonymousFunction",
+  "namedFunction.method",
+  "namedFunction.method.iteration.class",
```

Comment:
I think I would prefer to be more terse with this facet. Since there only is one iteration scope for a class in method I don't think we need to have class in the identifier itself.

Reply:
We discussed this at length but looks like we forgot to capture the outcome. Do you remember where we landed here?

Reply:
I think we landed in that we should just leave it as it is for now?

Reply:
Ok good well at least we have our conversation captured now 😂